pkgs/by-name/tu/turbo-unwrapped/package.nix

```diff
@@ -0,0 +1,100 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  protobuf,
+  rustPlatform,
+  autoPatchelfHook,
+  capnproto,
+  darwin,
+  extra-cmake-modules,
+  fontconfig,
+  lld,
+  # see comment below
+  # nix-update-script,
+  openssl,
+  pkg-config,
+  rust-jemalloc-sys,
+  zlib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "turbo-unwrapped";
+  version = "2.0.9";
+
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "v${version}";
+    hash = "sha256-hthuKGbHKTcv/wDIw9Ltcwl6jXWLhsCyqC76+vQyC5U=";
+  };
+
+  patches = [
+    # upstream uses nightly where lazy_cell is stable
+    ./enable-lazy_cell.patch
+  ];
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "pathfinder_simd-0.5.3" = "sha256-94/qS5d0UKYXAdx+Lswj6clOTuuK2yxqWuhpYZ8x1nI=";
+    };
+  };
+
+  nativeBuildInputs =
+    [
+      autoPatchelfHook
+      capnproto
+      extra-cmake-modules
+      pkg-config
+      protobuf
+    ]
+    # https://github.com/vercel/turbo/blob/ea740706e0592b3906ab34c7cfa1768daafc2a84/CONTRIBUTING.md#linux-dependencies
+    ++ lib.optional stdenv.isLinux lld;
+
+  buildInputs =
+    [
+      fontconfig
+      openssl
+      rust-jemalloc-sys
+      zlib
+    ]
+    ++ lib.optionals stdenv.isDarwin (
+      with darwin.apple_sdk_11_0.frameworks;
+      [
+        CoreFoundation
+        CoreServices
+        IOKit
+      ]
+    );
+
+  cargoBuildFlags = [
+    "--package"
+    "turbo"
+  ];
+
+  # Browser tests time out with chromium and google-chrome
+  doCheck = false;
+
+  env = {
+    # nightly features are used
+    RUSTC_BOOTSTRAP = 1;
+  };
+
+  passthru = {
+    # TODO: The repo's releases.atom doesn't seem to have any stable releases? This breaks updating for now :/
+    # updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "High-performance build system for JavaScript and TypeScript codebases";
+    homepage = "https://turbo.build/";
+    changelog = "https://github.com/vercel/turbo/releases/tag/v${version}";
+    license = lib.licenses.mpl20;
```

Comment:
```suggestion
    license = lib.licenses.mit;
```

```console
$ cat $(nix-build --no-out-link . -A turbo-unwrapped.src)/LICENSE
Copyright (c) 2024 Vercel, Inc

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

See #330110.